cas-server-support-saml/src/main/java/org/jasig/cas/support/saml/OpenSamlConfigBean.java

```diff
@@ -62,6 +65,7 @@ public ParserPool getParserPool() {
     @PostConstruct
     public void init() {
         LOGGER.debug("Initializing OpenSaml configuration...");
+        Assert.notNull(this.parserPool, "parserPool cannot be null");
```

Comment:
Why this `Assert.notNull` check as we already have a `@NotNull` for the `parserPool`?